app/models/disability_claim.rb

```diff
@@ -5,4 +5,9 @@ class DisabilityClaim < ActiveRecord::Base
   attr_accessor :successful_sync
 
   scope :for_user, ->(user) { where(user_uuid: user.uuid) }
+
+  def update_evss_data(raw_claim)
+    update_attributes(data: raw_claim, successful_sync: true)
+    touch # Ensure that updated_at is touched
```

Comment:
>  I don't think we want to change updated_at if the underlying data hasn't changed 

My understanding of https://github.com/department-of-veterans-affairs/sunsets-team/issues/117 was that we wanted to force an updated_at. If we want a separate column for last_access to use for TTL, we could change this to `touch :last_access`. I don't know if there's a way to consolidate DB calls though.